### ISSUE TITLE
Want to see profit both as % of this coin's trade, and also as % of total account value

### ISSUE THREAD

**Issue by libertyteeth** [taniman/profit-trailer#1539]
Enhancement request: in the pages for Pairs, DCA, and Pending, there's a "Profit" column.  This shows the percentage gain or loss for that coin's trade.  

I'd also like to see, perhaps underneath it (e.g., like it shows both Bid Price and Target Price in the same column, on the Pending Log page, just to the left of the Profit column), the "percentage gain or loss that this represents for the entire account".  

Especially on the DCA and Pending pages, since when it "doubles down" a few times, that coin would be a much greater percentage of the funds in the account.

**Comment by YoJeff** [comment]
Toss it over here for voting.
https://github.com/taniman/profit-trailer-enhancements/issues

**Comment by libertyteeth** [comment]
If I understand "Toss" correctly: your only action was to close this ticket.  You are requesting that I create a ticket in the profit-trailer-enhancements section.  (Also, I checked there, and don't see this issue.)

I will create a new ticket there, but wanted to leave this question: is it possible to move this issue to that other section?  I tried the "Edit" button on this ticket, but didn't see anything that looked like "move to other section".  Thanks!
